manifests/base/files/gen-istio-cluster.yaml

```diff
@@ -6024,6 +6025,15 @@ rules:
       - "certificatesigningrequests/approval"
       - "certificatesigningrequests/status"
     verbs: ["update", "create", "get", "delete", "watch"]
+  - apiGroups: ["certificates.k8s.io"]
+    resources:
+      - "signers"
+    resourceNames:
+    # Support legacy versions, before signerName was added
+    - "kubernetes.io/legacy-unknown"
+    - "istio.io/*"
```

Comment:
Does istio provide a signer?  KCM will not sign for anything outside of the kube signers (on purpose).

Reply:
Good point, we are not - we submit and approve the CSR, but we are not signing it. So this is probably not the right thing to put here. Source: https://github.com/istio/istio/blob/19bfdf0376ae6645c0c8f3fdee2cf2924995595c/security/pkg/k8s/chiron/utils.go#L67